CMakeLists.txt

```diff
@@ -234,10 +235,21 @@ else()
 				set(CMAKE_C_FLAGS "-O0 -g")
 			endif()
 		endif()
-		if(WANT_SANITIZER)
+		string(TOUPPER "${WANT_SANITIZER}" WANT_SANITIZER )
+		if(WANT_SANITIZER STREQUAL ADDRESS)
 			set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=undefined -fsanitize=address -fno-omit-frame-pointer")
-			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=undefined -fsanitize=address -fno-omit-frame-pointer")
-			set(CMAKE_STATUS_DEBUG_SUPPORT "Yes, with sanitizers")
+			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=undefined -fsanitize=memory -fsanitize=address -fno-omit-frame-pointer")
```

Comment:
undefined, memory, and address at the same time?

Reply:
Undefined and address can be used together. Memory is a copy/paste error, it can only be used alone.
I didn't notice because KVIrc only uses the c++ compiler, so only CXX_FLAGS was checked.
Fixed in https://github.com/kvirc/KVIrc/pull/2599/commits/38b91c6c9dfeb94a31e23c54b92011a3b2bba14c, thank you